tests/target/struct_lits_multiline.rs

```diff
@@ -107,3 +107,10 @@ fn issue201_2() {
         ..b
     };
 }
+
+fn issue491() {
+    Foo {
+        guard: None,
+        arm: 0, // Comment
+    };
+}
```

Comment:
could you add a test where the struct lit ends up on one line to check that the comment gets the `/* */` form? I.e., starts as `//`, but the struct lit gets compressed so it gets converted to `/* */`.